example_branches.json

```diff
@@ -0,0 +1,188 @@
+[
```

Comment:
Created this file to use incase we hit the github api limit too often while testing